### ISSUE TITLE
I'm getting the following error, although i has the sqlite3.dll (both x86 & x64) in my bin folder

### ISSUE THREAD

**Issue by ajaykoppisetty** [couchbase/couchbase-lite-net#472]
<log4net:exception>System.TypeInitializationException: The type initializer for 'Couchbase.Lite.Store.SqliteCouchStore' threw an exception. ---&gt; System.TypeInitializationException: The type initializer for 'NativeMethods' threw an exception. ---&gt; System.Exception: sqlite3.dll was not loaded.
   at SQLitePCL.SQLite3Provider.NativeMethods..cctor()
   --- End of inner exception stack trace ---
   at SQLitePCL.SQLite3Provider.NativeMethods.sqlite3_libversion()
   at SQLitePCL.SQLite3Provider.SQLitePCL.ISQLite3Provider.sqlite3_libversion()
   at Couchbase.Lite.Store.SqliteCouchStore..cctor() in c:\Users\Ajay\Downloads\AllInOne\dandeli\Couchbase\Couchbase.Lite.Shared\Store\SqliteCouchStore.cs:line 155
   --- End of inner exception stack trace ---
   at Couchbase.Lite.Store.SqliteCouchStore..ctor()
   at Couchbase.Lite.Database.Open() in c:\Users\Ajay\Downloads\AllInOne\dandeli\Couchbase\Couchbase.Lite.Shared\Database.cs:line 1965
   at Barrick.DataContext.IntializeCouchbase() in c:\Users\Ajay\Downloads\AllInOne\dandeli\Barrick\DataContext.cs:line 49
   at Barrick.Login.OnConnectedComplete(Boolean success) in c:\Users\Ajay\Downloads\AllInOne\dandeli\Barrick\Views\Login.xaml.cs:line 321
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()</log4net:exception></log4net:event>

**Comment by ajaykoppisetty** [comment]
We build an WPF app with the Couchbase lite and it was working fine on a windows 8.1 laptop but when we deployed the same on the windows 8.1 tablet, it was throwing the above error. What we have understood is, it is failing to create the couchbase lite files on the device. The product is about to release please help us ASAP in fixing the issue.

**Comment by borrrden** [comment]
This error happens when an appropriate C DLL cannot be found for sqlite.  Usually this error means it is not in the location expected (i.e. in the x86 or x64 folder next to SQLitePCL.raw.dll) but it can also mean that you've possibly included the wrong architecture type for your tablet.

**Comment by ajaykoppisetty** [comment]
Finally, we found the issue that was the msvcr110.dll missing. We added the respective architecture msvcr110.dll in the respective folders and it was working fine now. Thanks for the help :+1:

**Comment by borrrden** [comment]
@ajaykoppisetty I can't believe I forgot about that!  I just discovered the same issue for someone else a little while ago.  For everyone else who happens to find this, it means that the proper C++ runtime is missing.

**Comment by divvan** [comment]
I had the same exception on Windows 7 x64, despite having msvcr110.dll in both the System32 and SysWOW64 folders. 

It had an inner exception: "Could not load file or assembly 'SQLitePCL.raw, Version=0.8.1.0 ...". So I installed version 0.8.2 from NuGet, but then I got "System.IO.FileNotFoundException: Could not load file or assembly 'SQLitePCL.ugly, Version=0.0.0.0 ...". But after I installed version 0.8.2 of that, it worked.
